libcst/_nodes/base.py

```diff
@@ -32,6 +32,7 @@
 _T = TypeVar("_T")
 _CSTMetadataMapping = MutableMapping[Type["BaseMetaDataProvider[_T]"], _T]
 
+_CSTNodeT = TypeVar("_CSTNodeT", bound="CSTNode")
```

Comment:
We provides this type in _types to prevent duplication and circular dependency.
`from libcst._types import CSTNodeT`



from libcst._types import CSTNodeT
29




from libcst._types import CSTNodeT
29

Reply:
Oh, good idea!